mono/tests/test-driver

```diff
@@ -29,9 +29,10 @@ if ($test =~ /.*\|.*/) {
 	#This is a silly workaround, but all tests that use extra parameters need a larger timeout.
 	$timeout_in_minutes = 5;
 }
+$ENV{'TEST_DRIVER_TIMEOUT_SEC'} = $timeout_in_minutes * 60;
```

Comment:
So interestingly, the sgen tests always run with the Perl runner.  But I went ahead and added the env setting to the maanged runner too if it turns out useful in the future.